### PR TITLE
Updated CodeofConduct

### PR DESCRIPTION
**Pull request by creyente1897**
#### Describe the changes you have made in this pr -
Added ground rules and expectations in the code of conduct guide

### CODE REVIEW

File: CODE OF CONDUCT.md [hunk 1]
```diff
@@ -6,6 +6,13 @@ We are committed to making participation in this project a harassment-free exper
 
 Diversity and inclusion make the CircuitVerse community strong. We encourage participation from the most varied and diverse backgrounds possible and want to be very clear about where we stand.
 
+## Ground rules & expectations
+
+Before we get started, here are a few things we expect from you (and that you should expect from others):
+
+* Be kind and thoughtful in your conversations around this project. We all come from different backgrounds and projects, which means we likely have different perspectives on "how open source is done." Try to listen to others rather than convince them that your way is correct.
+* When adding content, please consider if it is widely valuable. Please don't add references or links to things you have created as others will do so if they appreciate it.
```
Comment (armudgal) [inline CODE OF CONDUCT.md]: Can you explain this point?

**Comment by armudgal** [comment]
Also pull from our `master` branch, since we renamed the code of conduct file.

File: CODE OF CONDUCT.md (hunk 1 above)
Comment (creyente1897) [reply CODE OF CONDUCT.md]: @armudgal I added it to let people know that there will always be different opinions for the same issue and we should respect each other opinions and users should add content to the guide if and only if it is valuable

**Comment by armudgal** [comment]
Hi, for the first point we have something already mentioned here https://github.com/CircuitVerse/CircuitVerse/blame/master/code-of-conduct.md#L9-L16 and for the second point it is mentioned here in a different language https://github.com/CircuitVerse/CircuitVerse/blame/master/CONTRIBUTING.md#L6-L17.

Worst case scenario: Someone makes a PR with the doc changes, if we think that the mention is worth it, the PR will get merged otherwise it will be closed with the satisfaction of the author.

Hence I think this PR is not needed. Thoughts?

**Comment by creyente1897** [comment]
@armudgal Yes you are right :smile:

**Pull request closed without merging**